python/Ganga/Runtime/GPIexport.py

```diff
@@ -18,6 +18,31 @@
 from Ganga.GPIDev.Base.Objects import GangaObject
 from inspect import isclass
 
+def _addToInterface(interface, name, _object):
+
+    if isType(_object, GangaObject):
+        setattr(interface, name, addProxy(_object))
+    elif isclass(_object) and issubclass(_object, GangaObject):
+        setattr(interface, name, getProxyClass(_object))
+    else:
+        setattr(interface, name, _object)
+
+def exportToInterface(myInterface, name, _object, doc_section, docstring=None):
```

Comment:
OK. To set in stone in this case are we populating both ```Ganga.GPI``` and ```ganga``` for the moment?

This does NOT expose all of the problems that __will__ come from 'turning ```Ganga.GPI``` off when we're ready to transition'.
There are a lot of subtle places (most of which I think I've fixed) where code is relying on old ```Ganga.GPI``` calls.

Why does the code need to be changed once we're using ```ganga``` the fact you can pass an interface isn't such a bad thing really is it?